editor/editor_settings.h

```diff
@@ -46,6 +46,7 @@ class EditorSettings : public Resource {
 	_THREAD_SAFE_CLASS_
 
 public:
+	static const String PROJECT_EDITOR_SETTINGS_PATH;
```

Comment:
I think you can make this `inline static const` and move the initialization here.

Reply:
Good idea, I didn't know this was possible.